package.json

```diff
@@ -65,7 +68,8 @@
     "prepare": "run-s clean build",
     "serve": "node server.js",
     "start": "run-p serve 'build -- --watch'",
-    "test": "jest",
+    "test": "jest && yarn test:types",
+	"test:types": "tsc -p tests/types",
```

Comment:
Am I correct that the purpose of this test to verify that `mergician.d.ts` will not break the TypeScript compiler? If so, the code contained within `tests/types/index.ts` is irrelevant and could be replaced with something as simple as the the following, yes?

```js
import mergician from 'mergician';

mergician({});
```